resources/views/components/input/password/password-strength.blade.php

```diff
@@ -0,0 +1,42 @@
+@slots(['label'])
+
+<div {{ $attributes->only('class')->class('') }}>
```

Comment:
Why do we need this? We can always just wrap our component within a div with the classes we need?